src/Components/Cart.jsx

```diff
@@ -1,8 +1,20 @@
-import { Dialog, Transition } from "@headlessui/react";
-import { XIcon } from "@heroicons/react/outline";
-import React, { Fragment } from "react";
+import { Dialog, Transition } from '@headlessui/react';
+import { XIcon } from '@heroicons/react/outline';
+import React, { Fragment, useEffect } from 'react';
+import { ShoppingCartIcon } from '@heroicons/react/outline';
 
 export default function Cart({ open, setOpen, cart, updateCart }) {
+  const displaySubtotal = () => {
+    let subtotal = 0;
+    cart.forEach((element) => {
+      // setCartSubTotal(cartsubtotal + element.price);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Remove commented code